tests/lib/rules/no-useless-flag.ts

```diff
@@ -872,9 +874,11 @@ describe("Don't conflict even if using the rules together.", () => {
             code: String.raw`/[\s\S]*/s;`,
             output: String.raw`/[\s\S]*/;`,
             rulesConfig: {
+                /* eslint-disable @typescript-eslint/naming-convention -- rule names */
```

Comment:
Most changes are related to this rule. Was this rule added to a recommended config or why does it cause so many errors now? It seems a little annoying...

Since this is a purely stylistic rule, we may even consider disabling the rule entirely.

Reply:
Thank you for checking this PR!
I will change `@typescript-eslint/naming-convention` to turn off.